confirm/confirm.go

```diff
@@ -0,0 +1,22 @@
+package confirm
+
+import (
+	"fmt"
+	"io"
+	"strings"
+)
+
+func ReadConfirm(r io.Reader) (bool, error) {
```

Comment:
```suggestion
func Read(r io.Reader) (bool, error) {
```

Would be too verbose otherwise `confirm.ReadConfirm(r)`.